corehq/apps/users/models.py

```diff
@@ -2001,6 +2007,39 @@ def get_location_map_case(self):
         except ResourceNotFound:
             return None
 
+    @property
+    @skippable_quickcache([], lambda _: settings.UNIT_TESTING)
```

Comment:
Correct, this will function as a singleton cache. You should add `'self._id'` as a vary on parameter.